docs/anonymous-statistics.md

```diff
@@ -55,12 +59,27 @@ Furthermore, the FATAL event sends the Netdata process & thread name, along with
 
 To see exactly what and how is collected, you can review the script template `daemon/anonymous-statistics.sh.in`. The template is converted to a bash script called `anonymous-statistics.sh`, installed under the Netdata `plugins directory`, which is usually `/usr/libexec/netdata/plugins.d`. 
 
-## Opt-Out
+## Opt-out
 
-To opt-out from sending anonymous statistics, you can create a file called `.opt-out-from-anonymous-statistics` under the user configuration directory (usually `/etc/netdata`). The effect of creating the file is the following:
+You can opt-out from sending anonymous statistics to Netdata through three different opt-out mechanisms:
 
--   The daemon will never execute the anonymous statistics script
--   The anonymous statistics script will exit immediately if called via any other way (e.g. shell)
--   The Google Tag Manager Javascript snippet will remain in the page, but the linked tag will not be fired. The effect is that no data will ever be sent to GA. 
+**Create a file called `.opt-out-from-anonymous-statistics`.** This empty file, stored in your Netdata configuration
+directory (usually `/etc/netdata`), immediately stops the statistics script from running, and works with any type of
+installation, including manual, offline, and macOS installations. Create the file by running `touch
+.opt-out-from-anonymous-statistics` from your Netdata configuration directory.
 
-You can also disable telemetry by passing the option `--disable-telemetry` to any of the installers.
+**Pass the option `--disable-telemetry` to any of the installer scripts in the [installation
+docs](../packaging/installer/README.md).** You can append this option during the initial installation or a manual
+update.
+
+**Set your `DO_NOT_TRACK` environmental variable to `1`.** You can set this variable with the following command: `export
```

Comment:
environmental variable -> environment variable (might be wrong but never seen it as environmental var.)